db.json

```diff
@@ -0,0 +1,496 @@
+            },
+            {
+              "name": "Domenic Denicola",
+              "email": "d@domenic.me"
+            },
+            {
+              "name": "吉野剛史 (Takeshi Yoshino)",
+              "email": null
+            }
+          ],
+          "reference": "STREAMS"
+        }
+      ]
+    },
+    {
+      "id": "url",
+      "name": "URL",
+      "scope": "URLs, their infrastructure, and their API.",
+      "editors": [
+        {
+          "name": "Anne van Kesteren",
+          "email": "annevk@annevk.nl"
+        }
+      ],
+      "standards": [
+        {
+          "name": "URL",
+          "href": "https://url.spec.whatwg.org/",
+          "description": "The URL Standard defines the infrastructure around URLs on the web.",
+          "authors": [
+            {
+              "name": "Anne van Kesteren",
+              "email": "annevk@annevk.nl"
+            }
+          ],
+          "reference": "URL"
+        }
+      ]
+    },
+    {
+      "id": "xhr",
+      "name": "XMLHttpRequest",
+      "scope": "A client-side networking API for transferring data between a web page and a server.",
+      "editors": [
+        {
+          "name": "Anne van Kesteren",
+          "email": "annevk@annevk.nl"
+        }
+      ],
+      "standards": [
+        {
+          "name": "XMLHttpRequest",
+          "href": "https://xhr.spec.whatwg.org/",
+          "description": "The XMLHttpRequest Standard defines the networking API exposed to scripts on the web.",
+          "authors": [
+            {
+              "name": "Anne van Kesteren",
+              "email": "annevk@annevk.nl"
+            }
+          ],
+          "reference": "XHR"
+        }
+      ]
+    }
+  ],
+  "ideas": [
+    {
+      "name": "CSS Books",
+      "href": "https://books.idea.whatwg.org/",
+      "description": "CSS Books defines CSS features for book publishing and high-quality printing of web pages.",
+      "authors": [
+        {
+          "name": "Håkon Wium Lie",
+          "email": null
+        }
+      ],
+      "reference": "BOOKS"
+    },
+    {
+      "name": "CSS Figures",
+      "href": "https://figures.idea.whatwg.org/",
+      "description": "CSS Figures defines CSS features for floating elements around pages and columns.",
+      "authors": [
+        {
+          "name": "Håkon Wium Lie",
+          "email": null
+        }
+      ],
+      "reference": "FIGURES"
+    }
+  ],
+  "biblio": [
+    {
+      "title": "Differences from HTML4",
+      "href": "https://html-differences.whatwg.org/",
+      "authors": [
+        {
+          "name": "Simon Pieters",
+          "email": null
+        }
+      ],
+      "reference": "DIFFERENCES"
+    },
+    {
+      "title": "DOM Parsing and Serialization Standard",
+      "href": "https://domparsing.spec.whatwg.org/",
+      "authors": [
+        {
+          "name": "Ms2ger",
+          "email": null
+        }
+      ],
+      "reference": "DOMPARSING",
+      "obsoletedBy": [
```

Comment:
Not sure if worth it but we could consider aligning on https://w3ctag.github.io/design-principles/#casing-rules by changing this to `obsoleted_by`. (I imagine we'd have to then change it back to `obsoletedBy` in biblio.json, or update the crawler that ingests biblio.json, so it feels kind of pointless.)